src/app/core/component/graph-display/graph-display.component.html

```diff
@@ -1,6 +1,39 @@
 <mat-sidenav-container [hasBackdrop]="false" class="container">
-    <mat-sidenav #start fixedInViewport="true" mode="over">
-        <h4>{{currentPattern?.name}}</h4>
+    <mat-sidenav *ngIf="patternView" style="min-width: 300px;" [disableClose]="true" [opened]="true" fixedInViewport="false" mode="push">
+        <h3>Add Patterns</h3>
+        <div *ngIf="allPatternsLoading; else content"
+             style="position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);">
+            <mat-spinner></mat-spinner>
+        </div>
+        <ng-template #content>
```

Comment:
Can you be a little more specific than naming it #content :) ? What is the content?